truffle/contracts/TuiChainController.sol

```diff
@@ -76,9 +74,10 @@ contract TuiChainController is Ownable
      *     atto-Dai per nano-Dai
      */
     function setMarketFee(uint256 _marketFeeAttoDaiPerNanoDai)
-        external onlyOwner
+        external
+        onlyOwner
     {
-        market.setFee({ _feeAttoDaiPerNanoDai: _marketFeeAttoDaiPerNanoDai });
+        market.setFee({_feeAttoDaiPerNanoDai: _marketFeeAttoDaiPerNanoDai});
```

Comment:
The space after a curly brackets it's good pratice too.

Reply:
I also couldn't find an option for this one (formatting for Solidity files uses https://github.com/prettier-solidity/prettier-plugin-solidity).